pages/page.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
We don't need shebangs in files that are not going to be executed directly on the command line.